src/posts/zkalc.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'zkalc: a cryptographic calculator'
+description: 'zkalc: a cryptographic calculator!'
+author: 'George Kadianakis, Michele Orrù'
+date: '2023-01-16'
+---
+
+<center>
+<a href="https://zka.lc">
+<img src="/images/posts/zkalc/zkalc_psychedelic.png" alt="psychedelic protocol descriptions with zkalc logo" width="700"/>
+</a>
+</center>
+
+# [zkalc](https://zka.lc) is a cryptographic calculator!
```

Comment:
`h1` is taken from `title` prop in he frontmatter, this won't render, so you can remove it